packages/datadog_flutter_plugin/ios/Classes/DatadogRumPlugin.swift

```diff
@@ -218,6 +218,17 @@ public class DatadogRumPlugin: NSObject, FlutterPlugin {
           FlutterError.missingParameter(methodName: call.method)
         )
       }
+    case "reportLongTask":
+        if let atTime = arguments["at"] as? NSNumber,
+           let duration = arguments["duration"] as? NSNumber {
+            let atDate = Date(timeIntervalSince1970: TimeInterval(atTime.doubleValue / 1000.0))
+            rum._internal?.addLongTask(at: atDate, duration: TimeInterval(duration.doubleValue / 1000.0))
```

Comment:
Same issue with the conversion here, you can use: 
```
TimeInterval(duration.doubleValue).toNanoseconds()
```

Reply:
Good catch on the Android ones! The math for this case is correct though, since TimeInterval is in seconds and the method here take's a TimeInterval, not a time in Nanoseconds.